packages/syft/src/syft/custom_worker/builder_k8s.py

```diff
@@ -0,0 +1,302 @@
+# stdlib
+from hashlib import sha256
+from pathlib import Path
+from time import time
+from typing import Dict
+from typing import Optional
+
+# third party
+import kr8s
+from kr8s.objects import APIObject
+from kr8s.objects import ConfigMap
+from kr8s.objects import Job
```

Comment:
should we name this something like `K8sJob` to make it not confused with the Job class we have in syft?

Reply:
Since the scope of this `Job` is limited to `builder_k8s`, it's completely fine to keep it as-is.